README.md

```diff
@@ -0,0 +1,224 @@
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 mvrva-transport ](#-mvrva-transport-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo) 
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [API Documentation](#documentation)
+    - [Run tests](#run-tests)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 mvrva-transport <a name="about-project"></a>
+
+This app is a desktop and mobile web application where a user can request for moving services by making a reservation for a particular service. The app has an admin section where the admin can add, edit and delete services. The app also has a user section where the user can view the services, make a reservation and delete that reservation.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org">React</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Users can sign up and login.
+- Services page after login.
+- Details page for a particular service.
+- A user can reserve a certain service
+- Admin and user functionality for the app.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Clone this repository to your desired folder:
+
+```
+git clone git@github.com:Al-Amin-Khan-Shakil/mvrva-transport-back-end.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Link to front end part of the site [FrontEnd](https://github.com/Al-Amin-Khan-Shakil/mvrva-transport-front-end/)
+- Ruby 3.3.0
+- Browser (Google Chrome, Mozilla Firefox, Safari or any other browser)
+- Run `Rails Server`
+
+### Setup
+
+Clone this repository to your dist folder:
+
+```
+git clone git@github.com:Al-Amin-Khan-Shakil/mvrva-transport-back-end.git
+cd mvrva-transport-back-end
+```
+
+### Install
+
+Install this project with:
+bundle install
+
+Create new master key and credential:
+Git Bash command: `EDITOR="code --wait", rails credentials:edit`
```

Comment:
- [ ] When I run this command according to the instructions, I get this:

![image](https://github.com/Al-Amin-Khan-Shakil/mvrva-transport-back-end/assets/57421823/54ddc1ae-f62c-451f-860e-1a7ac118ea97)

This is because the existing `credentials.yml.enc` must be deleted first.

Result:

![image](https://github.com/Al-Amin-Khan-Shakil/mvrva-transport-back-end/assets/57421823/6f9bb216-46d3-40a4-88c5-3ec44000118c)

Kindly, update your instructions to reflect this.

That way, all your users will be able to create their own master.key without errors. 👍